generator/src/main/java/com/scottlogic/deg/generator/GenerateExecute.java

```diff
@@ -40,7 +40,7 @@ public GenerateExecute(GenerationConfig config,
     @Override
     public void run() {
 
-        ValidationResult validationResult = validator.validateCommandLine(config);
+        ValidationResult validationResult = validator.validateCommandLine(config, fileOutputTarget);
```

Comment:
Should this be an argument to `validateCommandLine` or a dependency of  the `GenerationConfigValidator`?

Reply:
GenerationConfigValidator changes to use DI now.